bool all_plugins_running
bool all_plugins_stopped
